frontend/src/utils/recommendationsLanguages.ts

```diff
@@ -110,3 +91,40 @@ export const recommendationsLanguagesFromNavigator = (): string =>
         availableRecommendationsLanguages.includes(language)
       )
   ).join(',');
+
+export const loadRecoLanguagesFromLocalStorage = (
+  poll: string,
+  feed: string
+): string | null => storage?.getItem(`${poll}:${feed}:languages`) || null;
+
+export const saveRecoLanguagesToLocalStorage = (
+  poll: string,
+  feed: string,
+  value: string
+) => storage?.setItem(`${poll}:${feed}:languages`, value);
+
+/**
+ * Return the recommendations languages that should be used for anonymous and
+ * authenticated users that have not defined their preferred languages yet.
+ */
+export const initRecoLanguages = (): string => {
+  const languages = recommendationsLanguagesFromNavigator();
+
+  if (!someLangsAreSupported(languages.split(','))) {
+    return languages + `,${FALLBACK_LANG}`;
+  }
+
+  return languages;
+};
+
+/**
+ * Return the same languages as `initRecoLanguages`, but check the browser
+ * local storage first.
+ */
+export const initRecoLanguagesWithLocalStorage = (
+  poll: string,
+  feed: string
+): string => {
+  const languages = loadRecoLanguagesFromLocalStorage(poll, feed);
+  return languages === null ? initRecoLanguages() : languages;
+};
```

Comment:
The names of these functions does not make it clear when to use one or the other. Maybe they could be renamed to make the intention more explicit, like `getInitialRecoLanguages()` and `getInitialRecoLanguagesForFeed(poll, feed)` ?

Reply:
Yes, I agree. The difficulty comes from the feed For you. I didn't use `getInitialRecoLanguagesForFeed` because it may indicate that all feeds should use this method, but currently the feed For you doesn't. Only feeds that display a language filter benefit from `initRecoLanguagesWithLocalStorage`.

What do you think about the following names? I'll also update the documentation to describe the "why" and not the "how".
- `getInitialRecoLanguages`
- `getInitialRecoLanguagesForFilterableFeed(pool, feed)`

Reply:
Fixed by d6d35f6ab9cb3130a6b2cf7710b21d9d76e82ad1